common/djangoapps/third_party_auth/saml.py

```diff
@@ -247,8 +247,12 @@ def invalid_configuration(self):
             return missing
 
     def log_bizx_api_exception(self, transaction_data, err):
-        sys_msg = err.response.json() if err.response else 'Not available'
-        headers = err.response.headers if err.response else 'Not available'
+        try:
+            sys_msg = err.response.content
+            headers = err.response.headers
+        except AttributeError:
+            sys_msg = 'Not available'
+            headers = 'Not available'
```

Comment:
Maybe two separate `try/catch` blocks so you can still get one without the other?

Reply:
Good idea -- did that